helmexec/runner.go

```diff
@@ -15,10 +15,15 @@ type Runner interface {
 }
 
 // ShellRunner implemention for shell commands
-type ShellRunner struct{}
+type ShellRunner struct {
+	Dir string
+}
 
 // Execute a shell command
 func (shell ShellRunner) Execute(cmd string, args []string) ([]byte, error) {
 	preparedCmd := exec.Command(cmd, args...)
+	if shell.Dir != "" {
+		preparedCmd.Dir = shell.Dir
+	}
 	return preparedCmd.CombinedOutput()
```

Comment:
Are we sure we want to combine the stdout and stderr here? If my subcommand behaves correctly with regards to what it puts in stdout (meant to communicate info in a pipe) and stderr (meant for user info/errors), `CombinedOutput()` would break further pipeline building (as it would possibly interlace program ouptut and errors in the stdout).

Reply:
As this func isn't used in places that are relevant to piping, it would be ok. We do differente stdout/err in other places, like impl of the `exec` template func.

Reply:
What about piping helmfile output to other commands ?

Reply:
a possible usecase! however helmfile as of today emits too verbose logs, that only differentiating stdout/err here does't make sense. good for another issue though